models/src/controller.rs

```diff
@@ -290,3 +311,19 @@ pub fn controller_priority_class() -> PriorityClass {
         ..Default::default()
     }
 }
+
+/// Convert update time window values to cron expression
+/// For overnight schedule, we format it to `{}-23,0-{}` on hour spot.
+/// For example, start time on 18 and stop time on 5. we format it to `18-23,0-5`
+fn cron_expression_converter(update_window_start: String, update_window_stop: String) -> String {
```

Comment:
nit: Probably good to implement a `[test]` for this against the conditional logic